tsconfig.base.json

```diff
@@ -5,7 +5,10 @@
       // Allows for importing from `kibana` package for the exported types.
       "kibana": ["./kibana"],
       "kibana/public": ["src/core/public"],
-      "kibana/server": ["src/core/server"]
+      "kibana/server": ["src/core/server"],
+      "@emotion/core": [
+        "typings/@emotion"
+      ],
```

Comment:
Stubs the `@emotion/core` types, which are only used internally by Storybook, but cause type errors if included.

Reply:
Maybe Storybook should use a dedicated `tsconfig.json` instead of polluting the base global config?

Reply:
EUI PRs like this have highlighted the need to make better use of the various `tsconfig` files, especially for browser vs. node environments. @mistic and @spalger are aware, but I'm not sure if there is an issue tracking the idea.
Note that this is a temporary change that can be reverted when Storybook moves to Emotion 11. I'm happy to make further changes, though, if y'all decide that `kbn-storybook` should be handled differently.

Reply:
> I'm happy to make further changes, though, if y'all decide that kbn-storybook should be handled differently.

If @elastic/kibana-operations team thinks it's acceptable... @mistic any actionable items for the problem?